packages/react-components/src/atoms/Toggle.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import css from '@emotion/css';
+import { lineHeight, perRem } from '../pixels';
+import { steel, paper, tin, charcoal } from '../colors';
+const checkboxLabelStyles = css({
+  display: 'flex',
+  height: '54px',
+  borderRadius: '27px',
+  border: `1px solid ${steel.rgb}`,
+  overflow: 'hidden',
+  backgroundColor: steel.rgb,
+  color: tin.rgb,
+  g: {
```

Comment:
Hmm, this seems prone to break with changes of the `svg` as it's not colocated with the actual definition of the icon? Should we just have two icons?

Reply:
Maybe we could pass in props to the icon?

Reply:
Or does normal `fill: #color` do the job here?

Reply:
No it doesn't 👎